src/protocols/rdp/channels/rdpsnd/rdpsnd-messages.c

```diff
@@ -50,6 +50,14 @@ void guac_rdpsnd_formats_handler(guac_rdp_common_svc* svc,
     /* Reset own format count */
     rdpsnd->format_count = 0;
 
+    /* Check to make sure the stream has at least 20 bytes, which */
```

Comment:
Which what?

Reply:
It's a cliff-hanger.  Just channeling my inner novelist.